src/main/java/com/waluty/service/CurrencyServiceImp.java

```diff
@@ -0,0 +1,94 @@
+package com.waluty.service;
+
+import com.waluty.model.Currency;
+import com.waluty.model.dto.ConverterDto;
+import com.waluty.model.dto.CurrencyDto;
+import com.waluty.repository.CurrencyRepositoryImp;
+import lombok.AllArgsConstructor;
+
+import java.util.stream.Collectors;
+
+import java.util.*;
+
+@AllArgsConstructor
+public class CurrencyServiceImp implements CurrencyService {
+    private Map<String, List<Double>> comparedCurrencies = new LinkedHashMap<>();
+    private CurrencyDto currencyDto;
+    private ConverterDto converterDto;
+    private CurrencyRepository currencyRepository;
+    private Map<String, Double> differentMids = new LinkedHashMap<>();
+
+    private void setComparedCurrencies() {
+        List<Currency> temp = currencyDto.getAll();
```

Comment:
skad ta metoda wgl ci sie wziela getAll?!
jak juz:
List<Currency> temp = currencyRepository.getALL()
a prawidlowo pownni byc:
List<Currency> temp = new LinkedHashLis<>();
List<Currency> pom = currencyRepository.getAll();
i w petli
for(){
temp.add(converterDto.fromCurrencyToCurrencyDto(pom.get(i)));